resources/views/partials/alerts/success.blade.php

```diff
@@ -1,11 +1,6 @@
-<div class="row {{ $payload['classes'] }}" style="
-    padding: 15px;
-    color: #FFF;
-    background: #AAD26F;
-    border-radius: 5px;
-">
-    <div class="row__column row__column--compact">
-        <i class="fas fa-check"></i>
+<div class="row alert alert--success {{ $payload['classes'] }}">
+    <div class="row__column row__column--compact alert__fa">
+        <i class="fas fa-check fa-fw"></i>
     </div>
-    <div class="row__column text-center" style="font-weight: 600;">{{ $payload['message'] }}</div>
+    <div class="row__column text-center alert__message">{{ $payload['message'] }}</div>
```

Comment:
Same goes for these `row__column` and `text-center` classes. Stop using them here, and copy their logic into `alert__message`.